spec/player_input_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative "../lib/player_input"
+
+RSpec.describe PlayerInput do
+  describe "#get_coordinates" do
+    it "returns the user's input" do
```

Comment:
It would make more sense to me if the description said: "return the users's input if valid"